### PR TITLE
feat

### PR DESCRIPTION
**Pull request by nlok5923**
this pull request redirect the view route of three.js models of models selection page to GeoAr.js demo page.

### CODE REVIEW

File: public/js/dup-ar.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ function staticLoadPlaces() {
         {
             name: 'Pokèmon',
             location: {
-                // lat: <your-latitude>,
-                // lng: <your-longitude>,
+                lat: 22.7163,
+                lng: 75.8316,
```
Comment (nlok5923) [inline public/js/dup-ar.js:15]: @Naman-1234 please change it to your lat and long.

**Review by nlok5923 — COMMENTED** [review]
please do the suggested changes.

**Comment by nlok5923** [comment]
@Naman-1234 please review and let me know are these were the changes you demanded.

**Review by Naman-1234 — COMMENTED** [review]
Will be able to tell you after pulling and testing

**Comment by nlok5923** [comment]
@Naman-1234 haven't you pulled it yet.

**Pull request merged**